.github/workflows/apidocstest.yml

```diff
@@ -0,0 +1,105 @@
+name: api docs
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Would it make sense to have `api-docs/**` (like in https://github.com/getsentry/sentry/pull/21192/files) and `tests/apidocs/**` for both `on.push.paths` and `on.pull_request.paths`? Am I missing any other files involved?

Reply:
hmm maybe also `api/endpoints/**` since the tests are checking the responses against the schemas. What do you think @MeredithAnya and @NisanthanNanthakumar  ?

Reply:
hmm I don't think we should limit this test run by paths bc we are testing for the api contract to be the same between sentry API and the OpenAPI spec. Not too familiar, but couldn't a change to the serializers affect the resulting api?

Reply:
> since the tests are checking the responses against the schemas

That's what I was looking for, never mind then.